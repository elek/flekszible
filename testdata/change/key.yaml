apiVersion: apps/v1beta1
kind: StatefulSet
metadata:
  name: namenode
  annotations: {}
spec:
  serviceName: namenode
  replicas: 3
  template:
    metadata:
      labels:
        app: ozone
        component: namenode
    spec:
      containers:
      - name: namenode
        image: elek/ozone
        args: ["ozone","namenode"]
        env: []
