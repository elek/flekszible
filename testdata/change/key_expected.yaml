apiVersion: apps/v1beta1
kind: StatefulSet
metadata:
  name: datanode
  annotations: {}
spec:
  serviceName: prefix-datanode
  replicas: 3
  template:
    metadata:
      labels:
        app: ozone
        component: datanode
    spec:
      containers:
      - name: datanode
        image: elek/ozone
        args: ["ozone","datanode"]
        env: []
